includes/class-wc-privacy-erasers.php

```diff
@@ -287,6 +283,46 @@ public static function remove_order_personal_data( $order ) {
 
 		// Set all new props and persist the new data to the database.
 		$order->set_props( $anonymized_data );
+
+		// Remove meta data.
+		$meta_to_remove = apply_filters( 'woocommerce_privacy_remove_order_personal_data_meta', array(
+			'Payer first name'     => 'text',
+			'Payer last name'      => 'text',
+			'Payer PayPal address' => 'email',
+			'Transaction ID'       => 'numeric_id',
+		) );
+
+		if ( ! empty( $meta_to_remove ) && is_array( $meta_to_remove ) ) {
+			foreach ( $meta_to_remove as $meta_key => $data_type ) {
+				$value = $order->get_meta( $meta_key );
+
+				// If the value is empty, it does not need to be anonymized.
+				if ( empty( $value ) || empty( $data_type ) ) {
+					continue;
+				}
+
+				$anon_value = function_exists( 'wp_privacy_anonymize_data' ) ? wp_privacy_anonymize_data( $data_type, $value ) : '';
+
+				/**
+				 * Expose a way to control the anonymized value of a value via 3rd party code.
+				 *
+				 * @since 3.4.0
+				 * @param bool     $anonymized_data Value of this data after anonymization.
```

Comment:
This should be a string no? Also the variable name does not match the actual variable being used.